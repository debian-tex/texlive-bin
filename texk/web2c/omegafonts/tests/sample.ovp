(VTITLE JVF for SAMPLE)
(OFMLEVEL H 0)
(FONTDIR TL)
(FAMILY UNSPECIFIED)
(FACE F MRR)
(CODINGSCHEME UNSPECIFIED)
(DESIGNSIZE R 10.0)
(COMMENT DESIGNSIZE IS IN POINTS)
(COMMENT OTHER SIZES ARE MULTIPLES OF DESIGNSIZE)
(MAPFONT D 0
   (FONTNAME sample0-h)
   (FONTCHECKSUM H 11AB1731)
   (FONTAT R 1.0)
   (FONTDSIZE R 10.0)
   )
(MAPFONT D 2
   (FONTNAME sample1-h)
   (FONTCHECKSUM H 11AB1731)
   (FONTAT R 1.0)
   (FONTDSIZE R 10.0)
   )
(CHARACTER H 2121
   (CHARWD R 1.0)
   (MAP
      (SETCHAR H 0)
      )
   )
(CHARACTER H 2122
   (CHARWD R 1.0)
   (MAP
      (SELECTFONT D 2)
      (SETCHAR H 0)
      )
   )
