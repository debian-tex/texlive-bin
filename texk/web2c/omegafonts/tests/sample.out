(VTITLE JVF for SAMPLE)
(OFMLEVEL D 0)
(FONTDIR TL)
(FAMILY UNSPECIFIED)
(FACE F MRR)
(CODINGSCHEME UNSPECIFIED)
(DESIGNSIZE R 10.0)
(COMMENT DESIGNSIZE IS IN POINTS)
(COMMENT OTHER SIZES ARE MULTIPLES OF DESIGNSIZE)
(CHECKSUM H 0)
(SEVENBITSAFEFLAG FALSE)
(FONTDIMEN
   )
(MAPFONT D 0
   (FONTNAME sample0-h)
   (FONTCHECKSUM H 0)
   (FONTAT R 1.0)
   (FONTDSIZE R 10.0)
   )
(MAPFONT D 1
   (FONTNAME sample1-h)
   (FONTCHECKSUM H 0)
   (FONTAT R 1.0)
   (FONTDSIZE R 10.0)
   )
(COMMENT
   (CHARWD_ENTRY D 0 R 0.0)
   (CHARWD_ENTRY D 1 R 1.0)
   )
(COMMENT
   (CHARHT_ENTRY D 0 R 0.0)
   )
(COMMENT
   (CHARDP_ENTRY D 0 R 0.0)
   )
(COMMENT
   (CHARIC_ENTRY D 0 R 0.0)
   )
(CHARACTER H 2121
   (CHARWD R 1.0)
   (MAP
      (SETCHAR H 0)
      )
   )
(CHARACTER H 2122
   (CHARWD R 1.0)
   (MAP
      (SELECTFONT D 1)
      (SETCHAR H 0)
      )
   )
